.github/workflows/docker-build.yaml

```diff
@@ -570,12 +570,22 @@ jobs:
                     name: docker-compose-cypress
             -   name: Stop previous running application
                 run: docker compose down
+            -   name: Prepare docker-compose.yml file from docker-compose.github-actions.cypress.yml.dist
+                run: |
+                    sh .github/prepare-docker-compose-file.sh \
+                    "./docker/conf/docker-compose.github-actions.cypress.yml.dist" \
+                    "./docker-compose.yml" \
+                    "${{ needs.variables.outputs.DOCKER_PHP_FPM_BASE_REPOSITORY_TAG }}" \
+                    "${{ needs.variables.outputs.DOCKER_STOREFRONT_REPOSITORY_TAG }}" \
+                    "${{ needs.variables.outputs.DOCKER_ELASTICSEARCH_REPOSITORY_TAG }}"
             -   name: Build application
                 run: |
-                    mv docker-compose.cypress.yml docker-compose.yml
                     docker compose up -d --wait
-                    docker compose exec php-fpm php phing -D production.confirm.action=y db-create frontend-api-generate-new-keys build-demo-dev-quick error-pages-generate test-db-create test-db-demo test-elasticsearch-index-recreate test-elasticsearch-export
-                    docker compose exec php-fpm php phing -D production.confirm.action=y -D change.environment=acc environment-change
+                    docker cp ./ shopsys-framework-php-fpm:/var/www/html
```

Comment:
Why use this instead of volume?

Reply:
I can use the same docker-compose file then